core/src/libraries/constants/Constants.sol

```diff
@@ -126,11 +131,11 @@ uint256 constant FEE_GROWTH_AVG_WINDOW = 6 hours;
 /// @dev The length of the circular buffer that stores feeGrowthGlobals samples.
 /// Must be in interval
 /// \\( \left[ \frac{\text{FEE_GROWTH_AVG_WINDOW}}{\text{FEE_GROWTH_SAMPLE_PERIOD}}, 256 \right) \\)
-uint256 constant FEE_GROWTH_ARRAY_LENGTH = 48;
+uint256 constant FEE_GROWTH_ARRAY_LENGTH = 32;
 
 /// @dev The minimum number of seconds that must elapse before a new feeGrowthGlobals sample will be stored. This
 /// controls how often the oracle can update IV.
-uint256 constant FEE_GROWTH_SAMPLE_PERIOD = 15 minutes;
+uint256 constant FEE_GROWTH_SAMPLE_PERIOD = 1 hours;
```

Comment:
We'll be calling `update` _at most_ twice per day due to cost. Once to establish fee growth globals, and once to write the new IV to storage.

If I left this at 15 minutes, the maximum impact of that would be 0.04% (kinda useless). I think it's important to be able to keep IV somewhat up-to-date without spending too much on txn fees, so 1 hour is more effective.